connectivity/mbedtls/tests/TESTS/mbedtls/sanity/main.cpp

```diff
@@ -94,7 +113,7 @@ void test_crypto_asymmetric_encrypt_decrypt(void)
     unsigned char encrypted[64];
     unsigned char decrypted[sizeof(input)];
 
-    psa_set_key_usage_flags(&attributes, PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_ENCRYPT);
```

Comment:
This should be done in a separate commit.

Reply:
@Patater Fixed.